app/assets/stylesheets/components/_event.sass

```diff
@@ -0,0 +1,30 @@
+$Event-color-background: Theme-color(background, default)
+$Event-color-borderTop: #FF623F  // FIXME Put this color on background color MAP on theme.sass file.
+$Event-height: 150px
+$Event-spacing-base: $Theme-spacing-base
+$Event-text-title-color: Theme-color(text, default)
+$Event-width: 280px
+
+.Event
+  background-color: $Event-color-background
+  border-radius: 3px
+  border-top: 5px solid $Event-color-borderTop
+  box-shadow: 0 2px 10px -3px gray
+  display: inline-block
+  height: $Event-height
+  margin: $Event-spacing-base
+  padding: $Event-spacing-base
+  text-align: center
+  width: $Event-width
+  &:hover
+    background-color: lighten($Event-color-borderTop, 30%)
+
+.Event-info
+  list-style: none
+
+.Event-title
+  color: $Event-text-title-color
+  text-decoration: none
+
+.Event-create
```

Comment:
I don't think this is a good name, my idea was to have a separation from the events list, which is displaying the existing events from the left to the right. After that list, i think of place a button at the bottom. Yeah, but maybe it doesn't make sense for this to belong to this event, maybe on Body element.